specifyweb/frontend/js_src/lib/components/Merging/Header.tsx

```diff
@@ -132,16 +132,18 @@ function SummaryLines({
 export function MergeRow({
   header,
   children,
+  className = '',
 }: {
   readonly header: string;
+  readonly className?: string;
   readonly children: React.ReactNode;
 }): JSX.Element {
   return (
     <tr>
       <th
         className={`
           sticky left-0 text-left ${mergeCellBackground()} z-[10]
-        `}
+        ${className}`}
```

Comment:
can this be formatted nicely, pretty please? 🥺

Reply:
Fixed. Looked fine to me just fine before though ;)